common/djangoapps/entitlements/admin.py

```diff
@@ -65,6 +65,14 @@ class CourseEntitlementSupportDetailAdmin(admin.ModelAdmin):
     form = CourseEntitlementSupportDetailForm
 
 
+class CourseEntitlementPolicyForm(forms.ModelForm):
+    """Form for adding entitlement support details, exists mostly for testing purposes"""
```

Comment:
This seems like a copy paste comment?

Reply:
it one hundred percent is I was doing this so hurriedly last night. Thank you for catching my goof.